Gopkg.toml

```
@@ -30,6 +30,10 @@ ignored = ["google.golang.org/appengine*"]
   branch = "master"
   name = "golang.org/x/net"
 
+[[override]]
+  name = "github.com/gliderlabs/ssh"
+  revision = "d3a6756290240931faaf90ec97ddd5d9290623d8"
```

As lib use version can you use version ?
To update Gopkg.lock file use `dep ensure -update github.com/gliderlabs/ssh` this will not change much the file (normally just add the version line)
```suggestion
  revision = "d3a6756290240931faaf90ec97ddd5d9290623d8"
  version = "0.1.1"
```

---

Ah, good call. Sorry, I forgot I had been making specific changes to that lib.